docs/_connect-with-keycloak/05-add-identity-provider.md

```diff
@@ -47,22 +50,50 @@ If you click on the 'Identity providers' option (in the left-hand menu bar), you
 
 ![OpenID connect provider](/assets/images/connect-with-keycloak/keycloak-new-identity-provider.png)
 
-* When next signing in, you will see the option to sign in with 'oidc' (or whatever `alias` you chose to identify your identity provider).
+### Logging in with the new identity provider
```

Comment:
is this branch up to date? Thought this was in that other pr, could be wrong tho